src/js/game/states/play.js

```diff
@@ -84,25 +84,23 @@ play.create = function create() {
 
     this.game.camera.follow(this.disc, Phaser.Camera.FOLLOW_LOCKON, 0.1, 0.1);
 
-    this.bouncer = Bouncer.create({
-        actor: this.disc,
-        lower: 2,
-        upper: 10,
-    });
+    this.bouncer = Bouncer.create({ actor: this.disc });
 };
 
 play.update = function update() {
     const { disc, water, game } = this;
 
     this.bouncer.update(game.input.keyboard.isDown(Phaser.Keyboard.SPACEBAR));
 
-    game.physics.arcade.collide(disc, water, () => {
+    const didCollide = game.physics.arcade.collide(disc, water);
+    if (didCollide) {
         makeSplash(game, disc);
         this.bouncer.bounce();
-    });
+    }
```

Comment:
Why removing them from the callback ? 
Is there any benefits in terms of speed / precision ?